src/main.cpp

```diff
@@ -104,6 +104,7 @@
 #include "room_list.h"
 #include "game_loop.h"
 #include "music_player.h"
+#include "creature_senses.h"
```

Comment:
Perhaps this PR is a good one to move 'update_flames_nearest_thing' from main.cpp to sounds.c.

Reply:
Or maybe thing_objects.c?

Reply:
fine too I guess. It's a function that plays sounds for objects.